src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (nmbr1 = 0) => {
+    return (nmbr2 = 0) => {
+      count++;
+
+      const evenCalls = count > 3 && count % 2 === 0;
```

Comment:
do we mean the even calls here, or the break condition?
This variable may confuse, because from the first glance to me it seems that the robot should break on all the even calls